creates/newLinkPost.js

```diff
@@ -0,0 +1,75 @@
+const { API_BASE_URL } = require('../constants');
+
+// https://github.com/reddit-archive/reddit/wiki/API:-submit
+
+const parseResponse = ({ json: { success, jquery } }) => {
+  let message = jquery;
+
+  jquery.forEach((row, index) => {
+    if (row.length === 4 && (row[3] === 'text' || row[3] === 'redirect')) {
```

Comment:
In this type of  scenario would be great to add some comments on what these row elements are just for other engineers to understand what's going on here.

Reply:
I left the docs URL above the code, but I will always try to explain/expand my ideas in those cases